custom_components/gasbuddy/config_flow.py

```
@@ -305,7 +364,37 @@ async def async_step_manual(self, user_input=None):
+                        lat=self.hass.config.latitude,
+                        lon=self.hass.config.longitude,
+                        radius=100,
+                        limit=50,
+                    )
+                    if any(
+                        s["station_id"] == str(self._data[CONF_STATION_ID])
+                        for s in ev_res.get("stations", [])
+                    ):
+                        ev_charging = True
+                except Exception:  # noqa: BLE001
+                    pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)